server/routes/forms.ts

```diff
@@ -32,5 +36,23 @@ export default function routes({ dataFilterService }: Services): Router {
     })
   })
 
+  post('/github-repo-requests', async (req, res): Promise<void> => {
+    // const (github_repo,repo_description,base_template,github_project_visibility,jira_project_keys,product,github_projects_teams_admin,github_project_teams_writ,github_project_branch_protection_restricted_teams,slack_channel_release_notify,slack_channel_pipeline_notify,nonprod_alerts_severity_label,prod_alerts_severity_label) = req.body;
+    // const { github_repo } = req.body.github_repo
```

Comment:
Then you will get your form values here, these must match the `name` attribute specified on each form field.

Reply:
Just do a single field for now.